web/themes/joinup/joinup_theme.theme

```diff
@@ -948,8 +948,17 @@ function joinup_theme_preprocess_page_title__node__event(&$variables) {
   $field_event_date = $viewBuilder->viewField($node->get('field_event_date'), 'default');
   $field_event_location = $viewBuilder->viewField($node->get('field_location'), 'default');
 
+  $created = \Drupal::service('date.formatter')->format($node->getCreatedTime(), 'date_only');
+
+  /* @var \Drupal\Core\Entity\EntityViewBuilderInterface $viewBuilder */
```

Comment:
This part is repeated several times in different functions, it can be extracted into `_joinup_get_author()` function.